### PR TITLE
[oap-native-sql][CI] update travis for native sql

### PR DESCRIPTION
**Pull request by zhouyuan**
Signed-off-by: Yuan Zhou <yuan.zhou@intel.com>

## What changes were proposed in this pull request?
Update travis CI for native sql

## How was this patch tested?
travis

### CODE REVIEW

**Comment by zhouyuan** [comment]
@winningsix @zhixingheyi-tian travis seems not building for this patch, is there anything I need to change?

File: .travis.yml [hunk 1]
```diff
@@ -26,5 +26,36 @@ jobs:
       - cd ${TRAVIS_BUILD_DIR}/oap-cache/oap/
       - mvn clean -q -Ppersistent-memory test
       
+    - name: oap-native-sql
+      dist: bionic
+      jdk:
+        - oraclejdk8
```
Comment (offthewall123) [inline .travis.yml:32]: Travis works well now.Seems there exists grammar mistakes in your .travis.yml.
Why do you need to define jdk and dist here?

Comment (zhouyuan) [reply .travis.yml:32]: these should be supported in travis. 
https://docs.travis-ci.com/user/languages/java/
https://docs.travis-ci.com/user/languages/cpp/

To the question: recent cmake is required, arrow java need to be compiled with oracle jdk

File: .travis.yml [hunk 2]
```diff
@@ -26,5 +26,36 @@ jobs:
...
+      before_install:
+      echo $TRAVIS_COMMIT_MESSAGE
```
Comment (offthewall123) [inline .travis.yml:34]: add - here
echo $TRAVIS_COMMIT_MESSAGE

File: .travis.yml [hunk 3]
```diff
@@ -26,5 +26,36 @@ jobs:
...
+      - cd ../../java
+      - mvn clean install -q -P arrow-jni -am -Darrow.cpp.build.dir=/tmp/arrow/cpp/build/release/ -DskipTests -Dcheckstyle.skip
+      script:
+      - cd ${TRAVIS_BUILD_DIR}/oap-native-sql/
+      - cd cpp && mkdir build && cd build
+      - cmake .. && make
+      - cd ../../core
+      - mvn clean package -DskipTests #skip core tests
+      - cd ${TRAVIS_BUILD_DIR}/oap-data-source/arrow
+      - mvn clean -q tests
```
Comment (offthewall123) [inline .travis.yml:61]: delete empty lines

File: .travis.yml (hunk 2 above)
Comment (offthewall123) [reply .travis.yml:34]: - echo $TRAVIS_COMMIT_MESSAGE

File: .travis.yml (hunk 1 above)
Comment (offthewall123) [reply .travis.yml:32]: https://travis-ci.org/github/Intel-bigdata/OAP/jobs/689516068
Seems travis needs jdk9~15?

File: .travis.yml (hunk 2 above)
Comment (zhouyuan) [reply .travis.yml:34]: thanks, indeed it's the grammar error

**Pull request merged**